gravitee-apim-gateway/gravitee-apim-gateway-core/src/main/java/io/gravitee/gateway/jupiter/core/v4/invoker/EndpointInvoker.java

```diff
@@ -99,6 +100,13 @@ public Completable invoke(ExecutionContext ctx) {
                 );
             }
         }
+
+        try {
+            ctx.request().method(computeHttpMethodFromAttributes(ctx));
+        } catch (IllegalStateException e) {
+            return ctx.interruptWith(new ExecutionFailure(HttpStatusCode.BAD_REQUEST_400).key(INVALID_HTTP_METHOD).message(e.getMessage()));
+        }
```

Comment:
Rework this to avoid ugly try/catch

Waiting for https://gravitee.atlassian.net/browse/APIM-455 to be merged and base the refactor on this to have a more global approach.

The idea is to create a `computeRequest` method that would be enriched later on.
On this, we will set the method only if attribute exists in the context, and is valid.
If it's not valid, then just interrupt the chain.

Reply:
done.